README.md

```diff
@@ -64,8 +64,20 @@ You can customize the pre-defined lint sets, both to disable one or more of the
 lints included, or to add additional lints. For details see [customizing static
 analysis].
 
+## Migrating from the pedantic package
+
+In migrating from the pre-defined lint sets provided by the [pedantic package],
```

Comment:
Is this delta w/ the *latest* pedantic?  (I can't recall.)  If not, we should probably specify a version?